### ISSUE TITLE
[Feature] [video filters] AdvanceMAME2x [sf#87]

### ISSUE THREAD

**Issue by openMSX-import** [openMSX/openMSX#111]
**Reported by vlaflip on 2004-07-27 12:05 UTC**
AdvanceMAME2x produces a nice, not too blurry/ anti-
aliased picture and is, compared to others, speedy.
Would make a nice addition to the existing filters, as it 
would also run better on lower specs machines.

**Comment by openMSX-import** [comment]
**Commented by mthuurne on 2004-07-27 14:04 UTC**
Logged In: YES 
user_id=358343

I downloaded the AdvanceMAME source, but I cannot find this 
filter. I found hq2x, Scale2x and variations to them in 
"advance/blit", but we already support these, so that must not 
be what you mean. 

Could you provide some more info? Is "AdvanceMAME2x" actually 
a scaler, or is it a different kind of video filter? What name does 
AdvanceMAME itself use to describe it?

**Comment by openMSX-import** [comment]
**Updated by mthuurne on 2004-07-27 14:04 UTC**
- **assigned_to**: nobody --> mthuurne

**Comment by openMSX-import** [comment]
**Commented by vlaflip on 2004-07-27 15:10 UTC**
Logged In: YES 
user_id=1092216

*ahem* :P
I did some research, it seems AdvanceMAME2x was the old 
name for Scale2x..
Some apps call it AdvanceMAME2x, others AdvanceMAME 
Scale2x..

The Scale2x scaler was originally developed for the 
AdvanceMAME project.. hence the name.

Sorry for the useless submission.
Btw; I noticed openMSX is not listed in the list of projects at 
http://scale2x.sourceforge.net
I'll tell them about it ;)

**Comment by openMSX-import** [comment]
**Updated by vlaflip on 2004-07-27 15:10 UTC**
- **priority**: 5 --> 1

**Comment by openMSX-import** [comment]
**Updated by vlaflip on 2004-07-27 15:14 UTC**
- **status**: open --> closed

**Comment by openMSX-import** [comment]
**Commented by mthuurne on 2004-07-27 15:17 UTC**
Logged In: YES 
user_id=358343

No problem. I found that AdvanceMAME did some modifications 
to hq2x, I'm interested in testing whether those make it faster. 
Also they have a lot of MMX code that I can learn from. So your 
submission wasn't entirely useless :) 

I should tell them about it indeed. Manuel has been pushing me 
to do that for a while. I just have to make the time and describe 
the modifications I made, which might be interesting to the 
original authors.
